Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12-buster-slim
+FROM node:14-buster-slim
```

Comment:
I had trouble using that version. To validate run`docker build .`  It should fail complaining about python missing etc. To get around it I used:

FROM node:14-buster`
`